### ISSUE TITLE
Checkboxes on term page very faint with firefox mobile

### ISSUE THREAD

**Issue by simonpoole** [openstreetmap/openstreetmap-website#2302]
This seems to be a FF mobile specific issue, making the checkboxes essentially invisible (clicking on the text will improve things, but why would any body do that).

Not reproducible with Chrome and derivatives.

**Comment by tomhughes** [comment]
Well the site in general is shit on mobile so this is not really news...

**Comment by simonpoole** [comment]
While ist doesn't really work all too well in general, sign up is (outside of this and some other small issues not to bad). In any case I'll have a look at this when I back from vacation.

**Comment by tomhughes** [comment]
I'd look but I don't have a clue how to even start looking at something like this that only happens on mobile.

Surely it has to be a browser bug anyway?

**Comment by tomhughes** [comment]
Looks normal in desktop firefox with a phone sized viewport so I don't think it's just down to the small screen stylesheet.

**Comment by tomhughes** [comment]
I tried tweaking the colour and the border width and neither makes any difference.

**Comment by mmd-osm** [comment]
Is this on Android? I cannot see any difference between Firefox mobile and Chrome on iOS.

**Comment by tomhughes** [comment]
That's not really surprising as I believe Chrome and Firefox on iOS are really just UI shells - the actual browser engine is the Apple one because they don't allow anything else.

**Comment by simonpoole** [comment]
> Is this on Android?

Yes.

**Comment by simonpoole** [comment]
I did some research on this and I suspect it is something similar to this bug that has supposedly been fixed for three years

https://bugzilla.mozilla.org/show_bug.cgi?id=1352238

Note the effect I'm seeing also applies to the radio buttons above the CTs. googling finds a number of similar complaints but no resolution.

For documentation purposes, this is what it looks like (FF Android 68.8).

![Screenshot_20200507-123508_Firefox](https://user-images.githubusercontent.com/1011860/81288648-53306a80-9065-11ea-8131-3f6ae44bc1c5.jpg)

As there doesn't seem to be anything we can do about this, and I've fiddled around with the css too, I would suggest, outside of somebody having a brilliant idea, closing this.
